introduction.md

```diff
@@ -0,0 +1 @@
+Hi this is my introduction. I go by the handle of - Rockncoder.
```

Comment:
Looks good! Could you also please mention your favorite Code School path in your introduction? This could be Ruby, JavaScript, HTML & CSS, iOS or Electives.